examples/grpc-secured-unary/grpc_secured_unary_service.bal

```diff
@@ -1,15 +1,16 @@
 // This is the server implementation of the secured connection (HTTPS) scenario.
-import ballerina/config;
 import ballerina/grpc;
 import ballerina/log;
 
+// The configurable path to the Ballerina home.
+configurable string balHome = "<PATH_TO_THE_BALLERINA_HOME>";
```

Comment:
Do we need a default `Config.toml` file for the test execution to pass?

Reply:
@daneshk, what should be the better approach, 
- using the ballerina home path as a configurable variable OR
- use normal hard coded ballerina constant

Reply:
Here I added this hardcoded `balHome` configurable variable because, for the sake of completeness of this BBE, but don't want to go into much detail like `Config.toml`. There is no point in discussing configuration here since we have a separate BBE for that and we just want to emphasize the gRPC use case here.

Reply:
Shall we do it similar to https listener example. https://github.com/ballerina-platform/ballerina-distribution/blob/master/examples/https-listener/https_listener.bal#L11

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1255/commits/4af483997d546e843b1d8c51b23099ca04d205fc